src/main/java/org/threeten/extra/chrono/InternationalFixedDate.java

```diff
@@ -0,0 +1,887 @@
+     *
+     * @param zone the zone ID to use, not null
+     * @return the current date using the system clock, not null
+     */
+    public static InternationalFixedDate now(final ZoneId zone) {
+        return now(Clock.system(zone));
+    }
+
+    /**
+     * Obtains the current {@code InternationalFixedDate} from the specified clock.
+     * <p/>
+     * This will query the specified clock to obtain the current date - today.
+     * Using this method allows the use of an alternate clock for testing.
+     * The alternate clock may be introduced using {@linkplain Clock dependency injection}.
+     *
+     * @param clock the clock to use, not null
+     * @return the current date, not null
+     * @throws DateTimeException if the current date cannot be obtained
+     */
+    public static InternationalFixedDate now(final Clock clock) {
+        LocalDate now = LocalDate.now(clock);
+
+        return InternationalFixedDate.ofEpochDay(now.toEpochDay());
+    }
+
+    /**
+     * Obtains a {@code InternationalFixedDate} representing a date in the International fixed calendar
+     * system from the proleptic-year, month-of-year and day-of-month fields.
+     * <p/>
+     * This returns a {@code InternationalFixedDate} with the specified fields.
+     * The day must be valid for the year and month, otherwise an exception will be thrown.
+     *
+     * @param prolepticYear the International fixed proleptic-year
+     * @param month         the International fixed month-of-year, from 1 to 13
+     * @param dayOfMonth    the International fixed day-of-month, from 1 to 28
+     * @return the date in International fixed calendar system, not null
+     * @throws DateTimeException if the value of any field is out of range, or if the day-of-month is invalid for the month-year
+     */
+    public static InternationalFixedDate of(final int prolepticYear, final int month, final int dayOfMonth) {
+        return create(prolepticYear, month, dayOfMonth);
+    }
+
+    /**
+     * Obtains a {@code InternationalFixedDate} representing a date in the International fixed calendar
+     * system from the proleptic-year, for the month-less days of leap-day and year-day.
+     * <p/>
+     * This returns a {@code InternationalFixedDate} with the specified fields.
+     *
+     * @param prolepticYear the International fixed proleptic-year
+     * @param isLeapDay     is it leap-day, month-less day between end of June and beginning of Sol
+     * @param isYearDay     is it year-day, month-less day between end of December of the old year and beginning of January of the following year
+     * @return the date in International fixed calendar system, not null
+     * @throws DateTimeException if the value of any field is out of range,
+     *                           or if the day-of-month is invalid for the month-year
+     */
+    public static InternationalFixedDate of(final int prolepticYear, final boolean isLeapDay, final boolean isYearDay) {
```

Comment:
...What happens if you specify both fields?  It sounds like you should be able to specify that the day is _both_ the leap-day **and** the year-day, which is impossible.  The other reason I'd prefer a month/day-of-month set of flag values.